design_decisions.md

```diff
@@ -48,6 +48,10 @@ The data format should be arbitrary considering the API / data storage. On the c
 
 In an ideal world there would be a simple WYSIWYG editor which is stable and produces markdown output. However, after long discussions, we decided to use a WYSIWYG editor - namely CKEDITOR. We use the [Advanced Content Filter](http://sdk.ckeditor.com/samples/acf.html) to filter out invalid HTML tags. We are very strict on which tags can be used, and the goal is that we can switch between WYSIWYG editors freely.
 
+## Data-Upload
+
+For features like uploading images and use them on the web page, an upload service is needed. We decided to make use of the [Renuo-Upload](https://renuo.gitbooks.io/renuo-upload-doc/content/index.html) which is a service that we developed before.
```

Comment:
i would set the link of renuo-upload to the github repo?

you could describe it like that.
> Renuo Upload is a microservice which provides an easy way to upload and use uploaded files. Multiple files can be uploaded via drag and drop and can be stored on cloud-storages like Amazon S3.